app/models/host.rb

```diff
@@ -1103,7 +1103,12 @@ def refresh_services(ssu)
       services = MiqLinux::Utils.parse_chkconfig_list(services)
     end
 
-    services.each do |service|
+    containers = ssu.shell_exec("docker ps --format 'table {{.Names}}\t{{.Status}}' | tail -n +2")
+    if containers
+      containers = MiqLinux::Utils.parse_docker_ps_list(containers)
+    end
+
+    (services + containers).each do |service|
```

Comment:
Are docker containers the same thing as services?  This might be true for this specific openstack case but I doubt it is true universally for all host types.

IMO if openstack is returning some services as systemd units and some as docker containers this should be in the openstack host model.

We can split out a method here called `collect_services` which in openstack you could do basically `super.concat(docker_services)`